frog/imports/client/Wiki/index.js

```diff
@@ -654,13 +659,15 @@ class WikiComp extends React.Component<WikiCompPropsT, WikiCompStateT> {
       <div style={sideNavBarStyle}>
         <h2>{this.wikiId}</h2>
         <ul>
-          <Button
-            variant="contained"
-            color="primary"
-            onClick={() => this.setState({ createModalOpen: true })}
-          >
-            + Create new page
-          </Button>
+          {(!this.state.settings?.limitPageCreation || this.state.isOwner) && (
```

Comment:
As the two conditionals are not the same, but seem to be affecting the same behaviour, make sure that this is intentional, and consider moving this logic to a separate function to keep the code DRY and avoid inconsistencies.